javascript-client/test/api/ProjectApi.spec.js

```diff
@@ -56,45 +57,15 @@
       object[property] = value;
   }
 
-  function loginUser(email, password, done) {
-    const user = {
-      email: email.trim(),
-      password: password.trim()
-    }
-
-    User.findOne({ email: user.email }).then((savedUser) => {
-      if (!savedUser) {
-        const error = new Error('Incorrect email or password')
-        error.name = 'IncorrectCredentialsError'
-
-        return done(error)
-      }
-
-      const isMatch = savedUser.hashedPass === encryption.generateHashedPassword(savedUser.salt, password);
-
-      if (!isMatch) {
-        const error = new Error('Incorrect email or password')
-        error.name = 'IncorrectCredentialsError'
-
-        return done(error)
-      }
-
-      const payload = {
-        sub: savedUser.id,
-        role: savedUser.roles
-      }
-
-      // create a token string
-      const token = jwt.sign(payload, 's0m3 r4nd0m str1ng');
-
-      return done(null, token)
-    })
+  function setUserToken(userToken, bearer) {
```

Comment:
I would change this function to accept the clientInstance as aparameter. This way you will be able to move it in some common module and reuse it in all other tests.